src/Data/MinLen.hs

```diff
@@ -16,7 +16,7 @@ module Data.MinLen
     , TypeNat (..)
     , AddNat
     , MaxNat
-      -- * Minimum length newtype wrapper
+      -- * Minimum length data wrapper
```

Comment:
What's the motivation for this change? It seems like it's an accurate description to call it a newtype wrapper.